computingblog.py

```diff
@@ -71,4 +81,4 @@ def get_banner_image(filename):
     if len(sys.argv) > 1 and sys.argv[1] == "build":
         freezer.freeze()
     else:
-        app.run(ssl_context='adhoc')
```

Comment:
I didn't put this back, because I somewhat forgot.

Reply:
No worries. I see you have fixed it. Awesome!